hack/82410.patch

```diff
@@ -0,0 +1,3 @@
+diff --git a/vendor/k8s.io/code-generator/generate-groups.sh b/vendor/k8s.io/code-generator/generate-groups.sh
+old mode 100644
+new mode 100755
```

Comment:
Why is this necessary? Can we upstream it?

Reply:
good point, I didn't even realize this was a bug... https://github.com/golang/go/issues/34965